build.gradle.kts

```diff
@@ -13,6 +13,10 @@ repositories {
   mavenCentral()
 }
 
+dependencyCheck {
+  suppressionFiles.add("dependency-check-suppress-json.xml")
+}
+
```

Comment:
Realised that this needs adding for https://github.com/ministryofjustice/hmpps-notifications-alerts-vsip/pull/38 to work